.github/pull_request_template.md

```diff
@@ -24,9 +24,8 @@
     - [ ] will break the cluster.
           I.e. full cluster migration is required.
 
-**Pipeline config PR** *(if applicable)*
-<!-- https://github.com/elastisys/ck8s-pipelines/pull/ -->
-If you change some config options (e.g. rename variable or change the default value) we may need to update the pipeline.
+**Pipeline config** *(if applicable)*
+If you change some config options (e.g. rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.
```

Comment:
Maybe the case of adding a new variable should be mentioned here as well? I guess it's functionally pretty similar to renaming a variable, but seems good to be clear about it.